apps/react-18-tests-v8/tsconfig.cy.json

```diff
@@ -3,9 +3,13 @@
   "compilerOptions": {
     "jsx": "react",
     "isolatedModules": false,
-    "types": ["node", "cypress", "cypress-real-events"],
-    "typeRoots": ["../../node_modules", "../../node_modules/@types"],
+    "types": [],
     "lib": ["ES2019", "dom"]
   },
+  "files": [
```

Comment:
as we override the configs during type check, setting local typeRoots, simply referencing `symbol` would fail with resolution, thus moving to `files` dts inclusion